OSDM V4.0 draft: unable to validate the last draft with OpenapiGenerator

Hi OSDM team,

I'm currently trying to generate code from the OSDM 4.0 draft and it looks like the file is not well formed. 

I use OpenapiGenerator in this way: 

```bash
docker run --rm -v ${PWD}:/local openapitools/openapi-generator-cli validate -i /local/OSDM-online-api-v4.0.0-draft.yml
```

This give me a lots of errors:

````
Validating spec (/local/OSDM-online-api-v4.0.0-draft.yml)
[main] WARN  i.s.v.p.util.DeserializationUtils - Error snake-parsing yaml content
io.swagger.v3.parser.util.DeserializationUtils$SnakeException: Exception safe-checking yaml content  (maxDepth 2000, maxYamlAliasesForCollections 2147483647)
        at io.swagger.v3.parser.util.DeserializationUtils$CustomSnakeYamlConstructor.getSingleData(DeserializationUtils.java:483)
        at org.yaml.snakeyaml.Yaml.loadFromReader(Yaml.java:508)
        at org.yaml.snakeyaml.Yaml.load(Yaml.java:437)
        at io.swagger.v3.parser.util.DeserializationUtils.readYamlTree(DeserializationUtils.java:258)
        at io.swagger.v3.parser.util.DeserializationUtils.deserializeIntoTree(DeserializationUtils.java:172)
        at io.swagger.v3.parser.OpenAPIV3Parser.readContents(OpenAPIV3Parser.java:170)
        at io.swagger.v3.parser.OpenAPIV3Parser.readLocation(OpenAPIV3Parser.java:97)
        at io.swagger.parser.OpenAPIParser.readLocation(OpenAPIParser.java:16)
        at org.openapitools.codegen.cmd.Validate.execute(Validate.java:61)
        at org.openapitools.codegen.cmd.OpenApiGeneratorCommand.run(OpenApiGeneratorCommand.java:32)
        at org.openapitools.codegen.OpenAPIGenerator.main(OpenAPIGenerator.java:66)
Caused by: org.yaml.snakeyaml.scanner.ScannerException: while scanning for the next token
found character '\t(TAB)' that cannot start any token. (Do not use \t(TAB) for indentation)
 in 'string', line 6474, column 53:
     ... omponents/schemas/BookingMethod'           
                                         ^

        at org.yaml.snakeyaml.scanner.ScannerImpl.fetchMoreTokens(ScannerImpl.java:460)
        at org.yaml.snakeyaml.scanner.ScannerImpl.checkToken(ScannerImpl.java:239)
        at org.yaml.snakeyaml.parser.ParserImpl$ParseBlockMappingKey.produce(ParserImpl.java:637)
...
````

It turns out that a lots of <TAB> caracters are spread everywere in the file. Redocly dont mind this, but openapi-generator can not handle this.

Simply reformating the document using intellij fixes the problem.

I can provide a PR that 
- fixes the issues with formating
- add a verification step in github actions to check the validity of the draft file using openapi-generator.

Regards,
Sylvain

--- a/specification/v4.0/OSDM-online-api-v4.0.0-draft.yml
+++ b/specification/v4.0/OSDM-online-api-v4.0.0-draft.yml
@@ -6599,7 +6599,7 @@ components:
         bookingContact:
           $ref: '#/components/schemas/ContactDetail'
         bookingMethod:
-          $ref: '#/components/schemas/BookingMethod'	    
+          $ref: '#/components/schemas/BookingMethod'    
         bookingAccess:
           $ref: '#/components/schemas/BookingAccess'
         bookingAccessTime:
@@ -10959,7 +10959,7 @@ components:
         sections:
           type: array
           items:
-            $ref: '#/components/schemas/LegTrackSection'	  
+            $ref: '#/components/schemas/LegTrackSection'  
 
     LegTrackSection:
       type: object
@@ -10972,14 +10972,14 @@ components:
         positions:
           type: array
           items:
-            $ref: '#/components/schemas/GeoPosition'	  
+            $ref: '#/components/schemas/GeoPosition'  
         end:
-          $ref: '#/components/schemas/PlaceRef'	  
+          $ref: '#/components/schemas/PlaceRef'  
         duration:
           type: string
           format: duration
         distance:
-          $ref: '#/components/schemas/Distance'	 
+          $ref: '#/components/schemas/Distance' 
           minimum: 0
 
     Line:
@@ -14691,7 +14691,7 @@ components:
             $ref: '#/components/schemas/RegionalValidity'
           minItems: 1
         distance:
-          $ref: '#/components/schemas/Distance'	 
+          $ref: '#/components/schemas/Distance' 
 
     RegionalValidity:
       type: object
@@ -16740,7 +16740,7 @@ components:
         distance:
           description: |
             Distance in meters over the the complete trip, i.e., including transfer legs.
-          $ref: '#/components/schemas/Distance'	 
+          $ref: '#/components/schemas/Distance' 
         legs:
           description: |
             Legs ot the trip
@@ -17658,7 +17658,7 @@ components:
             - 'null'
           format: int32
         distance:
-          $ref: '#/components/schemas/Distance'	 
+          $ref: '#/components/schemas/Distance' 
 
     TripTravelAccountUnit:
       allOf:

